src/nft/components/layout/ExpandableText.tsx

```diff
@@ -0,0 +1,46 @@
+import clsx from 'clsx'
+import { useState } from 'react'
+
+import { Box, BoxProps } from '../Box'
+import * as styles from './ExpandableText.css'
+
+const RevealButton = (props: BoxProps) => (
+  <Box
+    as="button"
+    display="inline"
+    fontWeight="bold"
+    border="none"
+    fontSize="14"
+    color="darkGray"
+    padding="0"
+    background="transparent"
+    {...props}
+  />
+)
+
+export const ExpandableText = ({ children, ...props }: BoxProps) => {
+  const [isExpanded, setExpanded] = useState(false)
+
+  return (
+    <Box
+      display="flex"
+      flexDirection={isExpanded ? 'column' : 'row'}
+      alignItems={isExpanded ? 'flex-start' : 'flex-end'}
+      justifyContent="flex-start"
+      fontSize="14"
+      color="darkGray"
+      marginTop="0"
+      marginBottom="20"
+      {...props}
+    >
+      <span className={clsx(styles.span, !isExpanded && styles.hiddenText)}>
```

Comment:
i think this could just be set up as...
```ts
isExpanded ? <RevealButton>Show more</RevealButton> : <RevealButton>Show less</RevealButton>
```

with some additional props?